src/BasePlatform.ts

```diff
@@ -142,6 +142,15 @@ export default abstract class BasePlatform {
         return false;
     }
 
+    /**
+     * Returns true if the application currently has permission
+     * to display notifications. Otherwise false.
```

Comment:
the booleans are in the wrong order, which is why we try to name everything with positive intent. Can we rename this to `isDisturbable` or something for positive intent?